policybot/pkg/check/check.go

```diff
@@ -0,0 +1,324 @@
+/**
```

Comment:
Please add a copyright notice like in the other files.

Reply:
Let's call this package testresults or something. "check" is not descriptive of its function.